plume-admin-api-log/src/main/java/com/coreoz/plume/admin/LogEntryTransformer.java

```diff
@@ -45,24 +47,74 @@ default LogEntryTransformer applyOnlyToResponsesWithHeader(String headerName, St
         );
     }
 
+    /**
+     * This static method transforms the body of either the request or the response
+     * to be truncated to a given limit
+     * @param bodyCharLengthLimit : the limit of the body to be applied
+     * @return the corresponding {@link LogEntryTransformer}
+     */
     static LogEntryTransformer limitBodySizeTransformer(int bodyCharLengthLimit) {
-        return (request, response, apiTrace) -> {
+        return (request, response, apiLogEntry) -> {
             if (bodyCharLengthLimit < 0) {
-                return apiTrace;
+                return apiLogEntry;
             }
-            String bodyRequest = apiTrace.getBodyRequest();
+            String bodyRequest = apiLogEntry.getBodyRequest();
             if (bodyRequest != null && bodyRequest.length() > bodyCharLengthLimit) {
-                apiTrace.setBodyRequest(bodyRequest.substring(0, bodyCharLengthLimit));
+                apiLogEntry.setBodyRequest(bodyRequest.substring(0, bodyCharLengthLimit));
             }
-            String bodyResponse = apiTrace.getBodyResponse();
+            String bodyResponse = apiLogEntry.getBodyResponse();
             if (bodyResponse != null && bodyResponse.length() > bodyCharLengthLimit) {
-                apiTrace.setBodyResponse(bodyResponse.substring(0, bodyCharLengthLimit));
+                apiLogEntry.setBodyResponse(bodyResponse.substring(0, bodyCharLengthLimit));
             }
-            return apiTrace;
+            return apiLogEntry;
+        };
+    }
+
+    /**
+     * This static method transforms the body of either the request or the response
```

Comment:
Tu peux enlever `This static method ` pour juste commencer par : `Transforms the JSON body of either the request or the response`